apps/ledger-live-desktop/src/renderer/components/Web3AppWebview/NetworkError.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useCallback } from "react";
+import { Button, Flex, Icon, Link, Text } from "@ledgerhq/react-ui";
+import { useTranslation } from "react-i18next";
+import ErrorNoBorder from "~/renderer/icons/ErrorNoBorder";
+import { urls } from "~/config/urls";
+import { openURL } from "~/renderer/linking";
+import { languageSelector } from "~/renderer/reducers/settings";
+import { useSelector } from "react-redux";
+
+export const NetworkErrorScreen = ({ refresh }: { refresh: () => void }) => {
+  const { t } = useTranslation();
+  const locale = useSelector(languageSelector) || "en";
+
+  const handleContactSupport = useCallback(
+    () =>
+      openURL(
+        urls.contactSupportWebview[
+          locale in urls.contactSupportWebview ? (locale as keyof typeof urls.contactSupportWebview) : "en"
+        ],
+      ),
+    [locale],
+  );
+
+  return (
+    <Flex alignItems="center" justifyContent="center" flexGrow={1} flexDirection="column">
+      <ErrorNoBorder size={64} />
+      <Text
+        variant="h4Inter"
+        marginTop="16px"
+        fontSize={24}
+        fontWeight="semiBold"
+        fontFamily="Inter"
```

Comment:
[Question] Do we need to define Inter for both fontFamily and in the variant?

Reply:
I'm not entirely sure tbh.  IIRC it wasn't displaying with the correct font family unless I did this